doc/acceptance.md

```diff
@@ -0,0 +1,22 @@
+#Acceptance test 
+How to run the acceptance tests.
+
+## Preconditions
+
+### What you need on your computer to run the test
+- Retrieve the content of the __acceptance folder
+- Install all the requirements listed by the __acceptance/requirements.txt__ file: `pip install -r acceptance/requirements.txt` (we advise you to use a dedicated environment. Take a look at [virtualenv](https://pypi.python.org/pypi/virtualenv) for that).
```

Comment:
@gberche Currently I am using python 2.7. I did not  try with 3.X.
To install virtual env you can also lauch `sudo pip install virtualenv`
What you did by typing `virtualenv -p /usr/bin/python2.7 autosleepenv` force the version? Good to know.

You can also for your own need give a try to virtualenvwrapper (`pip install virtualenvwrapper`). The documentation is [here](https://pypi.python.org/pypi/virtualenvwrapper).

It allows you with some good export in your profile to launch commands such as `workon MyEnvironment` wich loads the environment, wherever you are. And there is completion on the environment name.